src/common/sidebar/components/Card.tsx

```diff
@@ -103,16 +122,36 @@ export function Card({ subject, title, code, selected, schedule, pid }: PropsWit
 
   return (
     <Box
-      bgColor={selected ? undefined : 'white'}
+      bgColor={
+        selected
+          ? inSessionSubject === undefined
+            ? undefined
+            : inSessionSubject
+            ? undefined
+            : '#d3d3d3'
+          : inSessionSubject === undefined
+          ? 'white'
+          : inSessionSubject
+          ? 'white'
+          : '#d3d3d3'
+      }
```

Comment:
This seems a bit extreme and could likely be simplified somehow. Or even just moving this into separate variables so it's easier to read would be great.